helmfile/charts/sc-logs-retention/scripts/logs-retention.sh

```diff
@@ -47,10 +47,23 @@ else
   for BACKUP in "${SC_LOG_BACKUPS_REST[@]}"; do
     if [[ ${S3_BACKUP} == "true" ]]; then
       BACKUP_URI="s3://${BUCKET_NAME}/logs/${BACKUP}"
-
+# As some providers have a fualty or incomplete implementation of the S3 API, a loop is needed in cases it's lacking
       echo "Deleting backup ${BACKUP_URI}"
-      aws s3 rm --recursive "${BACKUP_URI}" --endpoint-url="${S3_REGION_ENDPOINT}"
-    elif [[ ${GCS_BACKUP} == "true" ]]; then
+
+# Get the number of files currenctly avalible for deletion
+      NR_OF_FILES=$(aws s3 ls "${BACKUP_URI}" --endpoint-url="${S3_REGION_ENDPOINT}" | wc -l)
+# As long as their is files to be deleted, keep deleting them
+      while [ "$NR_OF_FILES" -gt 0 ]; do
+        aws s3 rm --recursive "${BACKUP_URI}" --endpoint-url="${S3_REGION_ENDPOINT}"
+# Wait 5 seconds to see if new files have been created while the deletion was done
```

Comment:
Indentation, same for all other comments as well
```suggestion
       # Wait 5 seconds to see if new files have been created while the deletion was done
```